src/Messaging/src/RabbitMQ/Exceptions/RabbitTimeoutException.cs

```diff
@@ -5,23 +5,22 @@
 using System;
 
 namespace Steeltoe.Messaging.RabbitMQ.
-    Exceptions
+    Exceptions;
```

Comment:
Can you make this a one-liner? It shows up as a diff after configuring my diff-tool to ignore the file-scoped namespace changes.